.travis.yml

```diff
@@ -41,7 +36,7 @@ before_install:
 
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda create -n testenv python=$TRAVIS_PYTHON_VERSION
+  - conda create -n testenv python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY
```

Comment:
Actually this was originally a mistake and I thought about removing it. But then I thought that may be it is a good idea to set up the environment with the correct version of ``numpy`` from the beginning. The command on line 59 will reinstall the correct version of ``numpy`` in case it was updated due to package dependencies.

Reply:
I think the original idea in #552 and its fix #554 was to use as little as possible from conda to potentially move to the standard python distribution provided by TravisCI, but (i) we have the xraylib dependency, which does not exist in PyPI, and (ii) as we also added the OSX support, we started relying on conda more, so it's OK to install it here with conda. I don't think we need to install it the second time with pip as pip will skip the installation in case numpy is already installed:
```bash
$ pip install numpy
Requirement already satisfied: numpy in /Users/mrakitin/miniconda3/envs/numpy/lib/python3.7/site-packages (1.14.6)
```
Do you mind removing the pip installation line?

Reply:
Done

Reply:
Thanks!